packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -4517,17 +4517,18 @@ Map {
       "collapsed": false,
       "content": undefined,
       "description": null,
+      "dynamicTransitionOffset": 48,
       "editable": false,
       "extraContent": undefined,
       "headerMode": "STATIC",
-      "headerModeDynamicOffSet": 0,
```

Comment:
This is technically a breaking change but since its unlikely anyone has picked up this feature we can let it slide. Let's try to avoid this in the future.